File: infrastructure/spring-based-app/work-log-web/src/app/model/worker.ts
export class Worker {

    login: string;
    name: string;
    surname: string;
    email: string;
    constructor(login?: string, name?: string, surname?: string, email?: string) {
        this.login = login;
        this.name = name;
        this.surname = surname;
        this.email = email;
    }
}

File: domain/src/main/java/pl/splaw/domain/model/Worker.java
package pl.splaw.domain.model;

import lombok.AllArgsConstructor;
import lombok.Getter;

/**
 *
 * @author Bartek <https://github.com/splaw88>
 */
@Getter
@AllArgsConstructor
public final class Worker {

    private final String login;
    private final String name;
    private final String surname;
    private final String email;

}
